src/validator.directive.js

```diff
@@ -201,6 +261,15 @@
     return {
       restrict: 'A',
       require: 'ngModel',
+      scope: {
+        model: '=ngModel',
+        initialValidity: '=initialValidity',
+        validCallback: '&',
+        invalidCallback: '&',
+        messageId: '@',
+        editReadonly: '=editReadonly',
+        originalData: '=originalData'
+      },
```

Comment:
We have put everything under link which allow us to isolate the scope when using multiple directive.